src/Tabs.jsx

```diff
@@ -63,9 +63,17 @@ export class Tabs extends React.Component {
 	}
 }
 Tabs.propTypes = {
-	children: React.PropTypes.arrayOf(
```

Comment:
I think the problem might be that `children` is not a true array, so when it tries to check the type of `children[0]`, it finds an object that is not the actual child.

That means that you can use `React.Children.forEach` in your custom function, but _also_ use `instanceof TabsTab`